.github/workflows/lint-test-sdk.yml

```diff
@@ -4,6 +4,7 @@ on:
   pull_request:
     paths:
       - '**/*'
+  workflow_dispatch:
```

Comment:
allows workflow to be triggered remotely via API

Reply:
I like the rename to be consistent with other SDKs as well 🧹